Lib/test/test_httplib.py

```diff
@@ -525,7 +525,8 @@ def test_dir_with_added_behavior_on_status(self):
         self.assertTrue({'description', 'name', 'phrase', 'value'} <= set(dir(HTTPStatus(404))))
 
     def test_simple_httpstatus(self):
-        class CheckedHTTPStatus(enum.IntEnum):
+        @enum._simple_enum(enum.IntEnum)
```

Comment:
I don't think that this change will be the proper approach to fixing the test suite issue.
see https://github.com/python/cpython/blob/53dceb53ade15587b9cfd30c0a0942232517dee9/Lib/enum.py#L1896-L1913

Reply:
I was wondering about the different use of enum in HTTPStatus and CheckedHTTPStatus, so I'm going to change it back.